aip/0100.md

```diff
@@ -309,8 +309,7 @@ CL, so that the history is preserved.
 
 Clearly document (using an internal comment in the proto) that you are
 violating an API design guideline and your rationale for doing so. This comment
-**must** be prefixed with go/not-api-precedent, in accordance with the guidance
-in that document.
+**must** be prefixed with aip.dev/not-precedent.
```

Comment:
Possibly put it in backticks? But definitely better than it was, either way.

Reply:
Done.